packages/angular_devkit/build_angular/src/angular-cli-files/utilities/check-port.ts

```diff
@@ -10,9 +10,9 @@ import { Observable } from 'rxjs';
 const portfinder = require('portfinder');
 
 
-export function checkPort(port: number, host: string, basePort = 49152): Observable<number> {
+export function checkPort(port: number, host: string): Observable<number> {
   return new Observable(obs => {
-    portfinder.basePort = basePort;
+    portfinder.highestPort = port;
```

Comment:
I think will not change the behavior, because basePort is never used when we set a ```port``` as option (https://github.com/indexzero/node-portfinder/blob/master/lib/portfinder.js#L108).

About the ```stopPort```, I just set the ```highestPort``` to maintain the code style. But, if you think will be more understandable set the ```stopPort``` instead, I can change the code.

Reply:
Indeed when the basePort will not be used and was not used. However, now introducing `stopPort` might have an effect because both start and stop ports will be 0.

Reply:
I have read again the code and you are right. Setting the ```stopPort``` the same as ```port``` will throw an error saying that there is no port available, which is not true.
Maybe set the ```stopPort``` as 49152, which was the previous value of ```basePort``` can be a better idea.

What do you think?

Reply:
Seeing the code, I don't see any benefits in using `stopPort/highestPort`

Reply:
Agree.